modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -179,7 +178,7 @@ EAL: No legacy callbacks, legacy socket not created
 ===========================================================
 Starting Test Suite : BBdev Validation Tests
 Test vector file = ldpc_dec_v7813.data
-Device 0 queue 16 setup failed
+Device 0 queue 16 setup failed <1>
```

Comment:
this should move to 

https://github.com/openshift/openshift-docs/pull/34331/files#diff-073d921c4422acd70e26398cd2409360d7307367df09da4ad91358823ba73ecdR203

line 203 on this file not here